go.mod

```diff
@@ -216,3 +216,5 @@ replace k8s.io/klog => k8s.io/klog v1.0.1-0.20200310124935-4ad0115ba9e4
 
 // bitbucket.org/ww/goautoneg has disappeard
 replace bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20190414153302-2ae31c8b6b30
+
+replace github.com/DataDog/agent-payload => github.com/DataDog/agent-payload v4.44.1-0.20201007091830-96613c2a95e4+incompatible
```

Comment:
Note: will bump this to the next release of agent-payload once the PR gets merged.